### ISSUE TITLE
How to please integrate InMobi (cordova-inmobi-sdk) into cordova application

### ISSUE THREAD

**Issue by zaka786khan** [Blah2014/phonegap-inmobi-plugin#1]
Blah2014! 

First of all thanks alot for developing such a nice plugin. Blah2014 I have problem integrating this plugin into my cordova project. Would you please help me how to configure this plugin and integrate into my cordova project. I mean where to add InMobi-4.x.x.jar file and how to add <application> tag as you know cordova rebuild .java files and MainActivity files and overwrites existings.

I think this is for phonegap not for cordova but as phonegap is on the top of cordova it should work on cordova.

In other words you can say I need information about how to integrate this plugin into cordova project if possible.

I need your help waiting for your kind response.

Thanks.

**Comment by Blah2014** [comment]
sure, no problem...do you have a Skype? if yes later tonight you can share your screen so i can help

Date: Wed, 16 Mar 2016 09:26:24 -0700

**Comment by zaka786khan** [comment]
yes yes I have skype and my id is zaka786khan please what is your skype I want to add you
I live in Pakistan and according to Pakistan I might be available through 5:00PM to 10:00PM when you want to contact to me and i am not available at skype you can send me sms on my number 0308-6112254 through send92.com website please I will be become available on skype.

Thanks

**Comment by Blah2014** [comment]
ddevayev
